test/fixtures/rules/css-modules/undefined.js

```diff
@@ -0,0 +1,15 @@
+// undefined.js
+import React from 'react';
+
+import styles from './test.css';
+
+// `css-modules/no-undef-class` - Alerts whenever a CSS class is not being used
+// ---------------------------------------------------------------------
+// Bad
+(function () {
+    const div = <div className={ styles.undefClass } />;
+})();
+// Good
+(function () {
+    const div = <div className={ [styles.usedClass, styles.unusedClass] } />;
```

Comment:
This is not valid JSX syntax. Either you use the `classnames` package or simply do the following, which doesn't require a new dependency:

```js
// Good
(function () {
    const div = <div className={ styles.usedClass } />;
    const div2 = <div className={ styles.unusedClass } />;
})();
```

Reply:
I didn't know it didn't work like this
fixed